custom/icds_reports/migrations/sql_templates/database_views/disha_indicators_v2.sql

```diff
@@ -0,0 +1,35 @@
+DROP VIEW IF EXISTS icds_disha_indicators_v2 CASCADE;
+CREATE VIEW icds_disha_indicators_v2 AS
+SELECT
+    "awc_monthly"."state_id" AS "state_id",
+    "awc_monthly"."state_name" AS "state_name",
+    "awc_monthly"."aggregation_level" AS "aggregation_level",
+    "awc_monthly"."month" AS "month",
+    "awc_monthly"."cases_household",
+    "awc_monthly"."cases_ccs_pregnant",
+    "awc_monthly"."cases_ccs_lactating",
+    "awc_monthly"."cases_child_health",
+    "awc_monthly"."num_launched_awcs",
+    "child_health"."ebf_in_month",
+    "child_health"."cf_in_month",
```

Comment:
how sure are you that this means the same thing as the tech spec asks. Specifically cf extends until the child is 3 years old while that asks for 6-8 months

Reply:
Hey, Because when these are told they are mapped with the existing indicators on the dashboard. So I just pulled those.
About cf_in_month stuff. yeah the requirement changed late night for cf. will be updating here it here